### ISSUE TITLE
Row append broken

### ISSUE THREAD

**Issue by ZohraS** [audreyt/ethercalc#386]
Hey guys,

I think you're starting to hate my guts, cause I can't stop finding new issues/questions to annoy you with.

This latest one comes from main.js, methinks. Basically, we were setting up our own server etc behind our vpn, and while that was being set up, we were working on a demo. The demo is much older than the new server (couple of months).

When we try to post a new row with the API, it works fine on the demo, but just doesn't work at all on the new server. The only things I've added to main.js were bits that we added in for our app to work as we want it to. And, the content of those is just copy/paste from the old version to the new one, without touching the rest of main.js at all. 

I swapped the new main.js for the old one, and suddenly, the POST works fine, and the line appends as it should.

I'll try to find some time to analyze and find the differences between the two versions, but thought I should give you a heads-up.

On another topic, Firefox keeps telling me that en-US.json is not found.  

Cheers!

**Comment by eddyparkinson** [comment]
>I swapped the new main.js for the old one

Which main? are you saying one of the main files in github has a bug, or the main that you created?

FYI  player.ls creates the client version of main.js

**Comment by ZohraS** [comment]
I realized that switching the new main.js with  the old main.js actually didn't make a difference.
I downgraded to an earlier version of ethercalc (ethercalc@0.20160501.1) , added my code, and suddenly it works again. :/

**Comment by audreyt** [comment]
Hi, sorry for the regression. Would you like to copy the URL request (in Chrome/Firefox inspector "Network tab" use "copy as cURL") and paste it here?

**Comment by ZohraS** [comment]
Hey,

Haven't slept yet, but I'll do that as soon as I can. Thanks!

**Comment by rugk** [comment]
I also see that Ethercalc tries to fetch the file `.../l10n/en-US.json`, which obviously does not exist.

```shell
$curl 'https://domain.example/CustomSubDir/l10n/en-US.json' --2.0 -H 'Host: domain.example' -H 'User-Agent: Mozilla/5.0 (X11; Linux x86_64; rv:49.0) Gecko/20100101 Firefox/49.0' -H 'Accept: */*' -H 'Accept-Language: en-US,en;q=0.5' --compressed -H 'X-Requested-With: XMLHttpRequest' -H 'DNT: 1' -H 'Connection: keep-alive' -H 'Cache-Control: max-age=0'
$curl 'https://domain.example/CustomSubDir/l10n/en.json' --2.0 -H 'Host: domain.example' -H 'User-Agent: Mozilla/5.0 (X11; Linux x86_64; rv:49.0) Gecko/20100101 Firefox/49.0' -H 'Accept: */*' -H 'Accept-Language: en-US,en;q=0.5' --compressed -H 'X-Requested-With: XMLHttpRequest' -H 'DNT: 1' -H 'Connection: keep-alive' -H 'If-Modified-Since: Tue, 17 May 2016 17:04:20 GMT' -H 'If-None-Match: W/"573b4f14-8f59"' -H 'Cache-Control: max-age=0' 
```

Version: https://github.com/audreyt/ethercalc/commit/2d598e222407474446739285deb6f796dfeb6c67

**Comment by audreyt** [comment]
The en-US issue is fixed in  575eea3; changed title to match. :)
